### DISCUSSION TITLE
Foggyforest2/Toxic Forest

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1127] [Maps and Content Showcase]
An updated version of that one foggy survival map, foggyforest, now on 1.9.9 .
Added a building and a bunch of other stuff compared to the original.
Day version has no volumetrics on it, but on the night version, it's used on the ground lamps, as it quite enhances the overall atmosphere.

Description : "One of the many Omega base installations, where they leave their toxic waste."

Day Version:
![20180121121636](https://user-images.githubusercontent.com/24359960/35195379-bd198076-fea9-11e7-9422-8e64335da804.png)
![20180121121644](https://user-images.githubusercontent.com/24359960/35195380-bd38bbc6-fea9-11e7-9b89-52cfa96f4e06.png)
![20180121124802](https://user-images.githubusercontent.com/24359960/35195381-bd549076-fea9-11e7-9064-3419e87fa8f8.png)
![20180121124824](https://user-images.githubusercontent.com/24359960/35195382-bd717c90-fea9-11e7-93ed-e746499dc9b4.png)

Night Version:
![20180121121553](https://user-images.githubusercontent.com/24359960/35195401-fab6f2d8-fea9-11e7-8b17-0deda02b61b9.png)
![20180121121626](https://user-images.githubusercontent.com/24359960/35195402-fad5e562-fea9-11e7-97ad-d6eb4f925ff9.png)
![20180121121401](https://user-images.githubusercontent.com/24359960/35195403-faf2b3ae-fea9-11e7-90fb-c68f4e651895.png)
![20180121121421](https://user-images.githubusercontent.com/24359960/35195404-fb0fa9b4-fea9-11e7-9d97-4496989aca60.png)
![20180121121529](https://user-images.githubusercontent.com/24359960/35195405-fb2e742a-fea9-11e7-8e5a-0136b85f20f9.png)

NOTE: Day version will probablly run slightly better for lower-end systems because of the lack of volumetric lights.

Edit1: I forgot to set the hurt mat on the water after changing water1 to water4, so that's fixed :p 

[foggyforest2.zip](https://github.com/red-eclipse/world/files/1667668/foggyforest2.zip)

**Comment by qreeves** [comment]
Very pretty, but I'm concerned about how open it is.

**Comment by no-lex** [comment]
- Trees are all the same size, color, height above ground, and orientation.
- Rocks are all the same size, color, height above ground, and orientation.
- Barrels are all oriented in the same direction and have the same pitch.
- Threshold for ramp to upper level is rather low.
- Ceiling lights are darker than the area the lights are actually lighting.
- Volumetric lights on the side of the building do not have shadows disabled, might help to set them to static since the only thing the lights entities can project shadows onto is the light textures themselves.
- Building still looks unfinished- could use some window/door thresholds, more finished flooring, and possibly some stuff inside to make it feel less empty.

![20180121223548](https://user-images.githubusercontent.com/23445929/35208181-c7f4b4b4-fefb-11e7-9e08-9b6546c196d0.png)
- I assume the whole set of consoles is intended to give off light, so it might be a good idea to make the lighting in here more diffuse. They probably can be set to nospec noshadow as well to save on performance.

![20180121223218](https://user-images.githubusercontent.com/23445929/35208226-0d67a98e-fefc-11e7-9886-d1731704efa5.png)
- Left a useless unaccessible area here.

- The night volumetrics look pretty nice, might as well enable them for daytime as well since you can toggle their availability in the settings menu anyway.
